fix for triangulation + qt5 upgrade

johndpope opened the pull request.

sujitkumardora committed:
Build Fix

sujitkumardora committed:
Build Fix

Commit:
WIP - attempt to upgrade qt4 -> qt5.
cmake -G Xcode

Commit:
update readme.

Commit:
fix readme.

johndpope committed:
Update README.md

johndpope commented:
DO NOT MERGE - until  I can resolve this error

ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

@sujitkumardora did you ever get this project working qt5?

sujitkumardora commented:
Hi @johndpope
The codes on [the repository](https://github.com/sujitkumardora/pixelstruct) (sujitkumardora/pixelstruct) is from the time when google code shutdown and to look through source I had to clone it to github. So its more or less abandoned at this point of time. I see [this repository](https://github.com/davidar/pixelstruct) (davidar/pixelstruct) just as old and looks abandoned as well. 

I am open to any suggestion for [the repository](https://github.com/sujitkumardora/pixelstruct) (sujitkumardora/pixelstruct)

Thank you

johndpope commented:
Hi @sujitkumardora , 

I think you'll find that @davidar is the owner of code. (even if abandoned. we'll see)
Copyright (c) 2009 David Roberts <d@vidr.cc>
any way - team work is the order of the day. Perhaps David could add you as a contributor to project? The microsoft photosynth service is shutting down - and I was reminded of the older photo-tourism style browsing they made in directx. Microsoft recently open sourced there latest viewer - but I'm particular interested in getting this project working - perhaps even porting to ios.
https://github.com/photosynth/offlineViewer/issues/1

 I've updated your code into a PR to fix deprecated triangulation stuff.

sujitkumardora commented:
Ok no problem I will just delete my repository as I no longer plan to use it in any way. 

Yes it was the microsoft photosynth that hocked me to look up this project.

Commit:
add pixelstruct.pro file
 update project to spit out Xcode project via qmake

Commit:
add .pro file

Commit:
Merge branch 'master' of github.com:johndpope/pixelstruct

Commit:
WIP - updates to readme

Commit:
stuck on x64 build failing.

Commit:
successfully building.
TODO - investigate this
[MainWindow::openImageDirectory] opening -NSDocumentRevisionsDebugMode
[BundleParser::parse] Unable to open file -NSDocumentRevisionsDebugMode/bundle/bundle.out

Commit:
add parameter to pass via edit schema

Commit:
if not images found - don’t exit.

johndpope commented:
working now on osx

Commit:
download  http://phototour.cs.washington.edu/datasets/NotreDame.zip - rename the notredame.out   bundle.out  and move a bundle folder.
update the passed parameter  /Users/johndpope/Downloads/NotreDame
